### PR TITLE
chore(deps): update go to 1.17

### PR DESCRIPTION
**Pull request by wass3r**
had some trouble with local integration test initially, but I think we're good now :)

### CODE REVIEW

**Review by delta592 — DISMISSED** [review]
looks good

**Review by jbrockopp — DISMISSED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022 Target Brands, Inc. All rights reserved.
+# Copyright (c) 2021 Target Brands, Inc. All rights reserved.
```
Comment (wass3r) [inline Makefile:1]: uh, not sure why this got reverted ? i will address in a follow-up commit - sorry

**Review by jbrockopp — DISMISSED** [review]
LGTM

**Comment by wass3r** [comment]
sorry, will need a re-review.. had a conflict to solve with main branch

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by kneal — APPROVED** [review]
LGTM 🐬

**Review by KellyMerrick — APPROVED** [review]
lgtm

**Pull request merged**